packages/next/test/fixtures/00-app-dir/vercel.json

```diff
@@ -2,7 +2,19 @@
   "builds": [
     {
       "src": "package.json",
-      "use": "@vercel/next"
+      "use": "@vercel/next",
+      "config": {
+        "functions": {
+          "app/api/**/*": {
```

Comment:
**question (non-blocking):** I'm just getting up to speed with the app directory and I know this is just a test, but I have a question about using this in my project. With `route.[tj]s`, you can have the API endpoint anywhere right? So I would want to do `"app/**/route.{ts,js}"` (or whatever the glob syntax is), right?

Reply:
@cb1kenobi yes, that should work too! As you said, wanted to make it a bit more explicit because of the test readability 👍